pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -261,7 +261,7 @@ self: super: {
         });
   aws-kinesis = dontCheck super.aws-kinesis;            # needs aws credentials for testing
   binary-protocol = dontCheck super.binary-protocol;    # http://hydra.cryp.to/build/499749/log/raw
-  binary-search = dontCheck super.binary-search;
+  binary-search = dontCheck (appendPatch super.binary-search ./patches/binary-search-fix-for-ghc-8.patch);
```

Comment:
Normally we try not to carry around patches here in nixpkgs.  Would it be possible to pull this patch from upstream using `fetchpatch`?  There should be examples of using `fetchpatch` in this file.

Reply:
Okay, i will do that, thanks for the suggestion

Reply:
Any progress?